src/main/webapp/itempagescript.js

```diff
@@ -143,16 +147,24 @@ function getItemPageComments(comments) {
  *
  * @param { string } comment - the comment including the username, message, and timestamp
  * @param { boolean } belongsToUser - boolean that shows whether or not the current user has posted a comment
+ * @param { string } email - the email of the user
```

Comment:
tiny nit: the current javadoc can be confused for the email of the current logged in user
```suggestion
 * @param { string } email - the email of the user who posted the comment
```